lib/rules/require-default-prop.js

```diff
@@ -141,11 +141,13 @@ module.exports = {
       const propsToReport = excludeBooleanProps(propsWithoutDefault)
 
       for (const prop of propsToReport) {
+        const propName = prop.propName != null ? prop.propName : `[${context.getSourceCode().getText(prop.key)}]`
```

Comment:
Why not `this.getStaticPropertyName(prop)`?

Reply:
I have been using `getStaticPropertyName` in the following:
https://github.com/vuejs/eslint-plugin-vue/pull/791/files#diff-eae83c5838640e1c85538d8c9fbe35e3R489